benefits/enrollment/templates/enrollment/success.html

```diff
@@ -52,18 +52,13 @@ <h2 class="h3 mt-lg-3 mb-1">{% translate "Your benefit will expire on" %} {{ enr
              alt="" />
       </div>
     </div>
-  </div>
-{% endblock inner-content %}
-
-{% block call-to-action %}
-  {% if authentication and authentication.logged_in and authentication.sign_out_button_template %}
-    <div class="row d-flex justify-content-start justify-content-lg-center">
-      <div class="col-12 col-lg-8 pt-lg-5 mt-lg-0 pt-4 mt-2">
-        <p>
+    {% if authentication and authentication.logged_in and authentication.sign_out_button_template %}
+      <div class="row justify-content-start justify-content-lg-center">
+        <p class="pt-lg-5 mt-lg-0 pt-4 mt-2">
           {% translate "If you are on a public or shared computer, don’t forget to sign out of " %}
           {% include authentication.sign_out_button_template %}
         </p>
       </div>
-    </div>
-  {% endif %}
-{% endblock call-to-action %}
```

Comment:
The current file has the `If you are on a public or shared computer..` line in the `call-to-action` area. But I moved it to the parent block container, `inner-content` instead. In the future, this will allow us to be more specific with what the `call-to-action` block has in it (if for example we ONLY use it for the majority case where there's just 1 big button).

Reply:
Ok so I think the idea here with moving it up to `inner-content` was so it can be inside the `<div>` defined on line 27 (`<div class="col-12 col-sm-12 col-lg-9">`) and therefore not have to duplicate the `col-` classes. 

Ultimately the spacing / alignment looks correct :white_check_mark: :

| Before | This branch |
| --- | --- |
| ![image](https://github.com/user-attachments/assets/a4f95b98-bb68-40f2-8135-b5cf90de053d) | ![image](https://github.com/user-attachments/assets/6cb6692f-954e-4feb-b0cd-e3ae6329891c) |


Could you explain more what you mean by:
> In the future, this will allow us to be more specific with what the `call-to-action` block has in it (if for example we ONLY use it for the majority case where there's just 1 big button). 

? I'm not fully following and am curious what it means to be more specific with the `call-to-action` block.

Reply:
Sorry that line was confusing 😆 , as I'm wondering to myself what it means 😆 
I'm gonna retract that statement entirely. https://github.com/cal-itp/benefits/pull/2583#discussion_r1878742909 achieved what I meant in that confusing sentence. 

Instead I'm going to say, although we _can_ put the `If you are on a public computer` sentence in _either_ `inner-content` or `call-to-action`, I think we should keep it in `inner-content` b/c it's not really the same design/intent as the `call-to-action` block, which is really meant to be the container of the big CTA buttons. So it's more of a semantics decision rather than a code change. 

Just in case in the future, there's some other new re-design to the `call-to-action` button / parent area, and we want to apply it to all the big buttons - it would be easier to do if we're only using this block *only* had the big buttons.